R/transformAbundance.R

```diff
@@ -0,0 +1,400 @@
+#' Transform Abundance
+#'
+#' This function applies transformation to abundance table, which is returned.
+#' Additionally, the information can be directly stored in the \code{assay}.
```

Comment:
Two function are discussed here. I think you can spell out, which function does the returning and which does the storing

Reply:
Added information

Reply:
I deleted those functions that only returned the transformed table.